src/components/_datepicker-demo/index.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from "react";
+
+import { registerLocale } from "react-datepicker";
+
+import DateTimePicker from "../DateTimePicker";
+import Custom from "./CustomInput";
+
+import enUs from "date-fns/locale/en-US";
+import ru from "date-fns/locale/ru";
+import fr from "date-fns/locale/fr";
+import hy from "../DateTimePicker/_helpers/custom_locale/hy";
+
+registerLocale("en", enUs);
+registerLocale("ru", ru);
+registerLocale("fr", fr);
+registerLocale("hy", hy);
+
+const Demo = props => {
+  const [date, setDate] = useState(null);
+  const [lang, setLang] = useState("en");
+
+  return (
+    <div style={{ padding: "20px" }}>
+      <div style={{ margin: "20px" }}>
+        <span>Language:</span>
+        <select style={{ fontSize: '16px' }} onInput={e => setLang(e.target.value)}>
+          <option value="en">English</option>
+          <option value="ru">Russian</option>
+          <option value="fr">French</option>
+          <option value="hy">Armenian</option>
+        </select>
+      </div>
+      <div style={{ margin: "20px" }}>
+        <DateTimePicker
+          date={date}
+          setDate={setDate}
+          dateFormat={[
+            "dd MMMM, yyyy",
+            "dd/MM/yyyy",
+            "dd.MM.yyyy",
+            "dd-MM-yyyy",
+            "dd/MM/yy",
+            "dd.MM.yy",
+            "dd-MM-yy"
+          ]}
+          customInput={Custom}
+          customInputProps={{ icon: "calendar" }}
+          locale={lang}
```

Comment:
I think this need to be moved into component itself, and locale should be picked up from navigator.

Reply:
yeah, now we import i18n instance from inside the DateTimePicker component, which gives access to chosen language.